config/routes.rb

```diff
@@ -148,6 +148,8 @@
         get :autocomplete, on: :collection
       end
 
+      get 'institutions/:id/children', to: 'institutions#children'
```

Comment:
Can you move this up a few lines inside the `resources :institutions` block?
` get :children, on: :member`